_includes/header.html

```diff
@@ -1,11 +1,11 @@
 <div class="header">
   <div class="container">
     <div class="heroText">
-      <h1>Painless REST backends</h1>
+      <h1>The Complete Application Stack</h1>
       <h2>
           <br/>
-          Build fully featured, <b>data driven</b> applications<br/>
-          with <b>objects</b> and <b>files</b> storage, <b>user authentication</b> and <b>push</b> notifications.
+            Build applications faster with <b>objects</b> and <b>files</b> storage,<br/>
```

Comment:
'file storage', not 'files storage'

Reply:
so `object` as well then `object and file storage`

Reply:
just did a commit to change it

Reply:
thanks!